### DISCUSSION TITLE
track casks for 1password7

### DISCUSSION THREAD

**Discussion by mlindgren80** [Homebrew/discussions#3240] [Casks]
Hello,

I recently noticed that the cask for 1password installs 8.x versions of 1password which require a subscription whereas 1password 7 can still run with a standalone license.  I notice that there were casks for 1password 6, but not 7.  Would it be possible to generate a 1password7 cask which tracks that particular release?  Thanks!

**Comment by miccal** [comment]
https://github.com/Homebrew/homebrew-cask-versions/pull/13829
